contract_api/consumers/service_event_consumer.py

```diff
@@ -112,6 +112,54 @@ def _get_new_assets_url(self, org_id, service_id, new_ipfs_data):
                                                                  service_id)
         return assets_url_mapping
 
+    def create_service_media(self,org_id,service_id,service_media,service_row_id):
+        count = 0;
+        if len(service_media)>0:
+            #clear the existing values from db
+            self._service_repository.delete_service_media(org_id=org_id,service_id=service_id)
+            #fif ipfs_url store in s3 and update url else store url
+            for service_media_item in service_media:
+                url = service_media_item.get("url",{})
+                if "http" in url or "https" in url:
+                    updated_url = url
+                    ipfs_url = ''
+                else:
+                    updated_url = self._push_asset_to_s3_using_hash(org_id=org_id,service_id=service_id,hash=url)
```

Comment:
Is this a synchronous operation? If so the lambda might timeout if a service has 20 images or videos. We need to test this. I suggest tracking this in a different issue